openstack/barbican/templates/etc/_ratelimit.yaml.tpl

```
@@ -5,6 +5,7 @@ whitelist:
 
 whitelist_users:
   - TS4_S4_SMTP_01
+  - ceph
```

it should be ceph-barbican: https://github.com/sapcc/helm-charts/blob/master/openstack/ceph-seed/templates/seed.yaml#L32
```suggestion
  - ceph-barbican
```

---

ah ok, presumed this first but wasnt sure.